apps/src/code-studio/pd/application/facilitator1819/Section1QuestionsAboutYou.jsx

```diff
@@ -118,5 +122,22 @@ export default class Section1QuestionsAboutYou extends Facilitator1819FormCompon
 
     return requiredFields;
   }
-}
 
+  /**
+   * @override
+   */
+  static getErrorMessages(data) {
+    const formatErrors = {};
+
+    if (data.zipCode && !ZIP_CODE_REGEX.exec(data.zipCode)) {
+      formatErrors.zipCode = "Must be a valid zip code";
+    }
+
+    if (!UsPhoneNumberInput.isValid(data.phone)) {
+      formatErrors.phone = "Must be a valid US phone number";
```

Comment:
Suggest making this "Must be a valid area code and phone number"

Also we should figure out if we are going to allow extensions

Reply:
"area code and phone number" or "10 digit phone number" ?

We may add extensions later, but it's lower priority.